src/main/java/org/socialhistoryservices/delivery/api/IISHRecordExtractor.java

```diff
@@ -47,4 +48,6 @@
      * @throws NoSuchPidException Thrown when the PID does not exist.
      */
     Map<String, ExternalHoldingInfo> getHoldingMetadata(Node node, String item) throws NoSuchPidException;
+
+    String getUnitIdsFromContainer(Node node, String pid, String container, boolean includeCurrentContainer) throws NoSuchPidException;
```

Comment:
This is very specific for EADs. Therefor this doesn't belong here in the interface for all records.